simple/src/constructorExample/ArgumentCons.java

```diff
@@ -0,0 +1,21 @@
+package constructorExample;
+
+public class ArgumentCons {
+
+	int a; // non static variable or global variable
+	static int b; // static variable
```

Comment:
please use public and refactor same in each and every file

Reply:
Done